Gemfile

```diff
@@ -11,18 +11,23 @@ gem 'bootsnap', require: false
 gem 'csv'
 gem 'datadog', '~> 2.0', require: 'datadog/auto_instrument'
 gem 'dogstatsd-ruby'
+gem 'dry-monads'
 gem 'flipper-active_record'
 gem 'health-monitor-rails'
 gem 'honeybadger'
 gem 'logstash-event'
 gem 'pg'
+gem 'rack'
 gem 'rack-cors'
 gem 'rack-utf8_sanitizer'
 gem 'rails_semantic_logger'
 gem 'railties', '~> 8.0'
 gem 'rake'
+gem 'rom-sql'
 gem 'rswag'
+gem 'sequel-activerecord_connection'
```

Comment:
This gem is fantastic -- it allows ROM.rb to use the same database connections and database transactions as activerecord, which kept our tests passing with no changes needed, even as they use a mix of ROM.rb and ActiveRecord.